Util/Helper.php

```diff
@@ -23,34 +24,31 @@ public function createTab(InputInterface $input, ?array $config = null): Tab
     {
         $tab = new Tab();
 
-        foreach($this->application->all() as $command) {
+        foreach ($this->application->all() as $command) {
             $commandInstance = $command instanceof LazyCommand
                 ? $command->getCommand()
                 : $command;
 
-            $annotations = $this->annotationReader->getClassAnnotations(new \ReflectionClass($commandInstance));
+            $reflectionClass = new \ReflectionClass($commandInstance);
 
-            foreach ($annotations as $annotation) {
-                if ($annotation instanceof Job) {
-                    $group = $input->hasOption('group') ? $input->getOption('group') : null;
+            if (PHP_MAJOR_VERSION >= 8.0) {
+                $attributes = $reflectionClass->getAttributes(Job::class);
 
-                    if ($group !== null && $group != $annotation->group) {
-                        continue;
+                if (count($attributes) > 0) {
+                    foreach ($attributes as $attribute) {
+                        $this->processJob(new Job(...$attribute->getArguments()), $input, $config, $commandInstance, $tab);
                     }
 
-                    $annotation->commandLine = sprintf(
-                        '%s %s %s',
-                        $config['php_binary'],
-                        realpath($_SERVER['argv'][0]),
-                        $annotation->commandLine === null ? $commandInstance->getName() : $annotation->commandLine
-                    );
+                    // Don't process annotations
+                    continue;
+                }
+            }
 
-                    if ($config['log_dir'] !== null && $annotation->logFile !== null) {
-                        $logDir = rtrim($config['log_dir'], '\\/');
-                        $annotation->logFile = $logDir . DIRECTORY_SEPARATOR . $annotation->logFile;
-                    }
+            $jobs = $this->annotationReader->getClassAnnotations($reflectionClass);
 
-                    $tab[] = $annotation;
+            foreach ($jobs as $job) {
+                if ($job instanceof Job) {
```

Comment:
I added a check if the result from classAnnotations because the result of getClassAnnotations can also contain commands containing other type of annotations, and will fail because of the type hinted property in the processJob method below (note that this also would fail prior to my PR)